src/components/organisms/WalletList/walletList.organism.tsx

```diff
@@ -0,0 +1,87 @@
+import React, {useEffect} from "react";
+import { useDispatch } from "react-redux";
+import {WalletConnectConnector} from "@web3-react/walletconnect-connector";
+import Typography from "../../atoms/Typography/typography.atom";
+import {
+  CONNECTOR_LABELS,
+  CONNECTOR_LOGOS,
+  CONNECTOR_DESCRIPTIONS,
+  supportedConnectorList,
+} from "../../../shared/constants/web3.constants";
+import { EColor, EFontWeight } from "../../../shared/types/styles.types";
+import {EConnectorType} from "../../../shared/types/web3.types";
+import styles from "./walletList.organism.module.scss";
+import WalletService from "../../../shared/services/wallet/wallet.service";
+import useWeb3 from "../../../hooks/useWeb3";
+import {EModalName} from "../../../redux/ui/ui.redux.types";
+import {toggleModal} from "../../../redux/ui/ui.redux.actions";
+import {EErrorTypes} from "../../../shared/types/error.types";
+
+const WalletList: React.FC = () => {
```

Comment:
Seems to me more like a Molecule than an organism, not being too complex but being opinionated. Wdy?

Reply:
This makes sense. My thinking here is WalletList as an organism as it renders a collection of entities (Wallet) and manages the state based on the events of the children (Wallet). Was planning to build the `WalletItem` as the molecule here. I can convert this to a molecule but would prefer to extract the `WalletItem` as the molecule component. Thoughts?

Reply:
> We can then make a WalletItem molecule, and since now WalletList would just basically .map() and render it doesn't really have a purpose anymore? In that case we can map in the v2 modal removing the need for the WalletList.organism

This sounds good. Since that modal is specific only to the wallet feature I can move the rendering there and remove the walletlist organism. Thanks!